### PR TITLE
Lock for Web versioning improvements and linting

### PR DESCRIPTION
**Pull request by jeffreylees**

### CODE REVIEW

File: articles/libraries/lock/v10/selecting-the-connection-for-multiple-logins.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Selecting the appropriate Identity Providers from multiple options is called "Ho
 
 ## Option 1: Programmatically
 
-When you initiate an authentication transaction with Auth0 you can optionally send a `connection` parameter. This value maps directly with any __connection__ defined in your [dashboard](${manage_url}).
+When you initiate an authentication transaction with Auth0 you can optionally send a `connection` parameter. This value maps directly with any __connection__ defined in your dashboard.
```
Comment (mpaktiti) [inline articles/libraries/lock/v10/selecting-the-connection-for-multiple-logins.md]: @jeffreylees why remove the dashboard link?

File: articles/libraries/lock/v9/index.md [hunk 2]
```diff
@@ -76,6 +75,28 @@ If you are using browserify to build your project, you will need to add the foll
...
+```
+
+### Parameters
+
+The possible parameters are detailed below.
+
+* **clientID {String}**: Your application clientID in Auth0.
+* **domain {String}**: Your Auth0 domain. Usually ```<account>.auth0.com```.
+* **options {Object}**:
+  * _**cdn {String}**_: Use as CDN base url. Defaults to `domain` if it doesn't equal `*.auth0.com`.
```
Comment (mpaktiti) [inline articles/libraries/lock/v9/index.md]: This doesn't look good, keep only the bold markdown ie `_**cdn {String}**_` -> `**cdn {String}**`

File: articles/libraries/lock/v9/selecting-the-connection-for-multiple-logins.md [hunk 3]
```diff
@@ -23,10 +22,11 @@ If using the [Lock](/lock), this is as simple as writing:
 
   auth0.show({connections: ['YOUR_CONNECTION']});
 
-
 Notice that this is equivalent of just navigating to:
 
-  https://${account.namespace}/authorize/?client_id=${account.clientId}&response_type=code&redirect_uri=${account.callback}&state=OPAQUE_VALUE&connection=YOUR_CONNECTION
+```text
+https://${account.namespace}/authorize/?client_id=${account.clientId}&response_type=code&redirect_uri=${account.callback}&state=OPAQUE_VALUE&connection=YOUR_CONNECTION
```
Comment (mpaktiti) [inline articles/libraries/lock/v9/selecting-the-connection-for-multiple-logins.md]: Add some whitespace so this is displayed better:

```text
https://${account.namespace}/authorize/?  
client_id=${account.clientId}&
response_type=code&
redirect_uri=${account.callback}&
state=OPAQUE_VALUE&
connection=YOUR_CONNECTION
```

File: articles/libraries/lock/v9/selecting-the-connection-for-multiple-logins.md [hunk 4]
```diff
@@ -23,10 +22,11 @@ If using the [Lock](/lock), this is as simple as writing:
 
   auth0.show({connections: ['YOUR_CONNECTION']});
```
Comment (mpaktiti) [inline articles/libraries/lock/v9/selecting-the-connection-for-multiple-logins.md]: Add some formatting here:

```text
auth0.show({connections: ['YOUR_CONNECTION']});
```

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Left some comments. I also saw some italics here and there, replace them with bold please.

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Looks good Jeff 👍  Resolve the conflict please and we are good to go.

**Comment by jeffreylees** [comment]
@mpaktiti done!

**Pull request merged**